src/node/internal/internal_zlib.ts

```diff
@@ -41,19 +285,6 @@ Object.defineProperties(
   )
 );
 
-export function crc32(
```

Comment:
I think we should move this function change to a different pull-request.

Reply:
Normally I'm in favour of splitting PRs into small chunks, however, this time I think the changes are OK to go together, because I'm reusing the input source stuff between zlibsync and crc32sync